### PR TITLE
SAR-7311: Updated a11y statement link for ALF

### PR DESCRIPTION
**Pull request by brennaksamuel**
!!! :guardsman: PLEASE UPDATE THE TEXT BELOW AND REMOVE THIS LINE :guardsman: !!!

# Descriptive Title - including Jira story or Github issue number

**Bug fix / New feature** (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs and context.

## Checklist

* [ ] I've included appropriate tests with any code I've added
* [x] I've executed the acceptance test pack locally to ensure there are no regressions
* [x] I've added my code using logical, atomic commits, squashing as appropriate - including the Jira issue number in the commit message
* [ ] I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/config/FrontendAppConfig.scala [hunk 1]
```diff
@@ -63,6 +63,9 @@ class FrontendAppConfig @Inject()(val servicesConfig: ServicesConfig, runModeCon
   lazy val reportAProblemNonJSUrl = s"$contactFrontendUrl/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
   lazy val betaFeedbackUrl = s"$contactFrontendUrl/contact/beta-feedback?service=$contactFormServiceIdentifier"
 
+  lazy val accessibilityStatementPath = loadConfig("accessibility-statement.host")
+  lazy val accessibilityStatementUrl = s"$accessibilityStatementPath" + "/accessibility-statement" + "/vat-registration"
```
Comment (palequezal) [inline app/config/FrontendAppConfig.scala:67]: You can just write this as s"$accessibilityStatementPath/accessibility-statement/vat-registration"

File: conf/application.conf [hunk 2]
```diff
@@ -314,7 +314,11 @@ controllers.test.TestSetupController = {
 
 noneOnsSicCodes = "OTk5OTksNzQ5OTA="
 
-accessibility-statement.service-path = "/vat-registration"
+accessibility-statement {
+    host = "http://localhost:12346"
+    path = "/accessibility-statement"
```
Comment (palequezal) [inline conf/application.conf:319]: Looks like you don't use path & service path - can be removed

File: app/config/FrontendAppConfig.scala (hunk 1 above)
Comment (brennaksamuel) [reply app/config/FrontendAppConfig.scala]: Oh yes that's true, I've updated it

File: conf/application.conf (hunk 2 above)
Comment (brennaksamuel) [reply conf/application.conf]: Updated

**Pull request merged**